If -i flag is passed, only export .gif file.

Aathish04 opened the pull request:
## List of Changes
- Add an option to the `open_file` method in `file_ops` which allows it to show the passed file in the OS's file explorer. 
- Make `SceneFileWriter` only export final video file if `write_to_movie` is True and  `save_as_gif` is False.
- Made the end "File Ready at:" message display the proper file path.
- Made `open_file_if_needed` in `__main__.py` only open "movie" files if `save_as_gif` is False and `write_to_movie` is True.
- Made `open_file_if_needed` in `__main__.py` use the `open_file` method in `file_ops`.

## Motivation
Fixes #267 

## Testing Status
Pytest exited with 0 errors.
Manually testing the flags yielded the required results.

## Further Comments
Currently, the option which makes manim open the file in the user's file explorer is named `-f` for `finder`. However, using it on a non-MacOS system will still work.
The current name might cause some confusion among end users, leading them to believe `-f` only works on MacOS.

Should the name of the flag be changed?

## Acknowledgement
- [x] I have read the [Contributing Guidelines](https://github.com/ManimCommunity/manim/wiki/Documentation-guidelines-(WIP))

leotrs commented:
> The current name might cause some confusion among end users, leading them to believe -f only works on MacOS.

Good catch. I think it's fine to keep the `-f`, but documentation should say that the `f` stands for `open a [f]ile`, rather than for `[f]inder`.

Aathish04 commented:
> EDIT: Having said that, the one change I would suggest is from `file_writer_config["show_file_in_finder"]` to just `file_writer_config["show_file"]` or something like that.

I've renamed it to `show_in_file_browser` in the latest commit, changed the flag to `--show_in_file_browser` and edited the help message for the flag as well.
